tests/scr/flatcar/build-shiftfs

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
We should parametrize and move this script to a centralized location to allow users to build shiftfs module by themselves -- something for later.

Reply:
Agreed ... future change.